stats.py

```diff
@@ -7,11 +9,12 @@ def scrape():
     enqueued_jobs = []
     workers = []
 
-    enqueued_jobs = [
-        dict(queue_name=q.name, size=q.count)
+    # Use chain to flatten the list of lists
+    enqueued_jobs = list(chain.from_iterable(
```

Comment:
Is the output still compatible with rq 0.x?

Reply:
I plan to create an additional commit to freeze the `rq` version to `1.6.1` (same version our `rq` is on). In that case, this will be compatable. Alternatively, I can conditionally add the `-failed` queues if the `rq` version is `1`+.

Reply:
Given that this is a general purpose exporter library, and is not tied to our code, it is better to support both 0.x and 1.x.

Reply:
Updated.